roles/taca/templates/site_taca_sarek_delivery.yml.j2

```diff
@@ -0,0 +1,61 @@
+log:
+    file: "{{ ngi_pipeline_site_path }}/log/taca.log"
+
+deliver:
+    rootpath: "{{ proj_root }}/{{ ngi_pipeline_site_delivery }}/nobackup/NGI"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    operator: "{{ recipient_mail }}"
+    hash_algorithm: md5
+    files_to_deliver:
+{% if "upps" == site %}
```

Comment:
I guess this means that Sthlm has a separate command for staging reports for WGS deliveries, right?

Reply:
Yeah, in the current state. But I'm waiting for some input from Sthlm.

Reply:
:+1: